cmd/virt-handler/virt-handler.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+
+# Configure libvirt username and password
+sed -i s/^authname=/authname=$(cat /etc/sasl-secret/username)/g /etc/libvirt/auth.conf
+sed -i s/^password=/password=$(cat /etc/sasl-secret/password)/g /etc/libvirt/auth.conf
+
+# Start virt-handler
+/virt-handler -v 3 --libvirt-uri qemu:///system --hostname-override $NODE_NAME
```

Comment:
Don't put these parameters here - leave them in the k8s manifest, as we need to be able to customize tem without rebuilding the image.